session6/src/homework/StatementEx.java

```diff
@@ -0,0 +1,17 @@
+package homework;
+
+public class StatementEx {
+    public static void main(String[] args) {
+
+        int a = 20;
+        if (a>=18) {
+            System.out.println("I am adult");
+        }
+
+        else {
```

Comment:
cool, but please format the code and try to teach yourself to write well formatted code. In order to be compliant with Java Language Specification, this `else` keyword should be on the same line with the ending of `if` block, meaning after `}` at line 9.
To format the code with Eclipse Keymap: CTRL + Shit + F
or, with mouse clicks in IntelliJ -> top left hamburger menu -> Code -> Reformat Code.